### DISCUSSION TITLE
Graphics heavy books

### DISCUSSION THREAD

**Discussion by mand331** [apprenticeharper/DeDRM_tools#1624] [General]
Children's books frequently are graphics heavy content and converting them from Azw or Kfx to Epub and then viewing them in a epub reader, e.g., in Moon+ usually results in a less than ideal experience. The graphics don't fill the page, the graphics may be out of line with the text, etc. I have converted many text books to Epub without problem. But, these graphics heavy books frequently do not seem to convert nicely. Does anyone have any tips or point me to resources for settings or readers that work best? Thanks in advance.

**Comment by ElleKayEm** [comment]
Try looking over at the MobileRead.com calibre forums.
